MetaMask/metamask-extension
# Swaps: Create a new swap

## Explanation
As a user, I want to submit multiple swaps in a row without clicking on multiple buttons to reset the flow.

We've also changed the button to `Close` after swapping is completed, which redirects a user to the main page.

## Manual testing steps

Make another swap:
- Make a swap
- On the success page click on `Create a new swap`
- The `Build Quote` page is displayed again

Close button:
- Make a swap
- Click on the `Close` button
- The main page is displayed

## Screenshots
Success page with the `Create a new swap` link and `Close` button:
<img width="355" alt="image" src="https://user-images.githubusercontent.com/80175477/118670758-304ba100-b7f7-11eb-8950-a7827c553cb8.png">
